.github/workflows/generalized-deployments-reproducible.yaml

```diff
@@ -0,0 +1,23 @@
+on: push
+
+name: Generalized Reproducible Deployments
+jobs:
+  push:
+    name: Invoke General Reproducible Docker Build Pipeline
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.GDBP_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.GDBP_AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-west-2
+    - name: Override GITHUB_REF and thus ECR destination for master
+      run: echo "GITHUB_REF_OVERRIDE=refs/heads/dev" >> $GITHUB_ENV
+      if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
s/master/main/

Unfortunately I haven't found a variable for the default branch in github actions yaml.

Reply:
Good catch. Fixed in e21ab31.